spec/core/instances_spec.rb

```diff
@@ -545,4 +553,100 @@
       end
     end
   end
+
+  describe '#has_string_attribute?' do
+    it 'returns false if no string attribute exists' do
+      expect(subject.has_string_attribute?).to be false
+    end
+
+    context 'dataset has string attribute' do
```

Comment:
I usually try to write context + it descriptions, so that you can read them as sentence if they are combined.
Would it make sense to adjust the context description to sth. like `'if dataset has string attribute'`?